packages/preset-create-react-app/index.d.ts

```diff
@@ -0,0 +1,15 @@
+import { Configuration, RuleSetCondition } from 'webpack';
+
+interface Options {
```

Comment:
Nice. Do we need these for the other presets too?

Reply:
We seem to have them for some... I think we could move the whole project to TS after we get this out, and then we won't need this.